### PR TITLE
Add eks-rbac module

### PR DESCRIPTION
**Pull request by spartan-tainguyen**
## Summary
Add new module AWS/eks-rbac to setup Kubernetes RBAC roles/clusterroles and map with AWS IAM.
## Types of Changes

- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 🕷 Bug fix (non-breaking change which fixes an issue)
- [ ] 👏 Performance optimization (non-breaking change which addresses a performance issue)
- [ ] 🛠 Refactor (non-breaking change which does not change existing behavior or add new functionality)
- [ ] 📗 Library update (non-breaking change that will update one or more libraries to newer versions)
- [x] 📝 Documentation (non-breaking change that doesn't change code behavior, can skip testing)
- [ ] ✅ Test (non-breaking change related to testing)
- [ ] 🔒 Security awareness (changes that effect permission scope, security scenarios)

### CODE REVIEW

File: aws/eks-rbac/.gitignore
Comment (spartan-ytruong) [inline aws/eks-rbac/.gitignore:1]: should we remove this file a?

File: aws/eks-rbac/variables.tf
Comment (spartan-ytruong) [inline aws/eks-rbac/variables.tf:1]: please format and remove redundant EOL

File: aws/eks-rbac/.gitignore
Comment (spartan-tainguyen) [reply aws/eks-rbac/.gitignore:1]: I added this file by mistake. I have removed it, thank you.

File: aws/eks-rbac/variables.tf
Comment (spartan-tainguyen) [reply aws/eks-rbac/variables.tf:1]: Do you mean to add the missing EOL? I have added it, thanks

**Review by spartan-ytruong — APPROVED** [review]
lgtm

**Pull request merged**